src/plugins/intel_cpu/src/transformation_pipeline.cpp

```diff
@@ -243,6 +244,7 @@ void Transformations::PreLpt(const std::vector<ov::element::Type>& defaultPrecis
         manager.register_pass<ngraph::pass::low_precision::ConvertSubtractConstant>(defaultPrecisions);
     }
     manager.register_pass<ov::pass::Validate>();
+    manager.register_pass<RefConvertI64ToI32>();
```

Comment:
Can we place the new transformation before Validate pass?

Reply:
Validate removes this Convert node and pass ConvertPrecision does not work properly.